packages/web-api/src/WebClient.spec.ts

```diff
@@ -964,6 +971,76 @@ describe('WebClient', () => {
     });
   });
 
+  describe('requestInterceptor', () => {
+    it('can intercept out going requests', async () => {
```

Comment:
Thanks for adding tests! Let's clarify the intention of this test by expanding on the test title. For someone unfamiliar with request interceptors in axios, what does 'intercepting' a request do? Anyone can figure out that the `requestInterceptor` intercepts requests, but to what to end? Asked differently: what is this test asserting?